src/tests/ftest/soak/soak_harassers.yaml

```diff
@@ -18,28 +24,48 @@ setup:
     start_servers: True
     start_agents: True
 server_config:
+    engines_per_host: 2
     name: daos_server
     provider: ofi+verbs;ofi_rxm
     nr_hugepages: 8192
     control_log_mask: INFO
     control_log_file: daos_control0.log
     servers:
-        log_mask: ERR
-        fabric_iface: ib0
-        fabric_iface_port: 31416
-        log_mask: ERR
-        log_file: daos_io0.log
-        # Storage definitions for AEP and NVMe
-        scm_mount: /mnt/daos0
-        scm_class: dcpm
-        scm_list: [/dev/pmem0]
-        bdev_class: nvme
-        bdev_list: ["aaaa:aa:aa.a","bbbb:bb:bb.b"]
+        0:
+            targets: 4
+            pinned_numa_node: 0
+            nr_xs_helpers: 1
+            fabric_iface: ib0
+            fabric_iface_port: 31317
+            log_file: daos_server0.log
+            bdev_class: nvme
+            bdev_list: ["aaaa:aa:aa.a","bbbb:bb:bb.b"]
+            scm_class: dcpm
+            scm_list: ["/dev/pmem0"]
+            scm_mount: /mnt/daos0
+            log_mask: ERR
+            env_vars:
+                - FI_UNIVERSE_SIZE=2048
+        1:
+            targets: 4
+            pinned_numa_node: 1
+            nr_xs_helpers: 1
+            fabric_iface: ib1
+            fabric_iface_port: 31417
+            log_file: daos_server1.log
+            bdev_class: nvme
+            bdev_list: ["cccc:cc:cc.c","dddd:dd:dd.d"]
+            scm_class: dcpm
+            scm_list: ["/dev/pmem1"]
+            scm_mount: /mnt/daos1
+            log_mask: ERR
+            env_vars:
+                - FI_UNIVERSE_SIZE=2048
     transport_config:
-      allow_insecure: True
+        allow_insecure: True
 agent_config:
     transport_config:
-      allow_insecure: True
+        allow_insecure: True
```

Comment:
This test should be running with secure mode,
please use allow_insecure: False
Unless there is any failure.

Reply:
enabled secure mode for harasser; 48h and smoke.   Did not enable secure mode for 2h test because that is run on A21 and servers are already configured and are not started by avocado.